### DISCUSSION TITLE
Why does this abstract to GTK rather than Qt?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [picoe/Eto#2371] [Q&A] [answered]
Surely Qt, because it supports so many more platforms, is the superior choice?

**Comment by cwensley** [comment, marked as answer]
There was/is no _good_ Qt bindings for .NET afaik.  I'd welcome someone contributing a Qt platform but it certainly isn't a priority.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/picoe/Eto/discussions/2371#discussioncomment-4584693

Yeah, @cwensley, that's what I found online too. In fact, I barely found any decent .NET interfaces.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/picoe/Eto/discussions/2371#discussioncomment-4632876

@cwensley, have you seen https://github.com/qt-labs/qtdotnet/blob/251910aacf46f2c97c7d7f80adabd47d8d23b819/README.md#qt--net?
